include/openssl/rsa.h

```diff
@@ -152,8 +152,9 @@ OPENSSL_EXPORT void RSA_get0_crt_params(const RSA *rsa, const BIGNUM **out_dmp1,
 // |n|, |e|, and |d| respectively, if non-NULL. On success, it takes ownership
 // of each argument and returns one. Otherwise, it returns zero.
 //
-// |d| may be NULL, but |n| and |e| must either be non-NULL or already
-// configured on |rsa|.
+// For a public key, |d| may be NULL, but |n| and |e| must either be non-NULL
```

Comment:
Call out the special missing `e` thing is for JCA only...

Reply:
Are we sure that JCA is the only entity (or language ecosystem) that allows private keys to be specified without `e`? I'm a little hesitant to over-specify the comment if it's a more general usage pattern. Although, for what it's worth, I did to a quick spot-check of [golang's](https://pkg.go.dev/crypto/rsa#PrivateKey) and [pyca's](https://cryptography.io/en/latest/hazmat/primitives/asymmetric/rsa/#cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateNumbers) analogous structures, and both seem to expect `e`. so maybe it is just a Java thing...

Reply:
i'll update the comment.